code/UI/General/TabMenus/Scoreboard/Entry/ScoreboardEntry.scss

```diff
@@ -54,3 +60,39 @@ scoreboardentry
         border-bottom-right-radius: 4px;
     }
 }
+
+.tag-buttons-div
+{
+    display: flex;
+    justify-content: center;
+
+    &.no-display
+    {
+        display: none;
+    }
+}
+
+.Friend
```

Comment:
let's make these lowercase, it's just apart of our styling. You could make the strings in the list lowercase, then just use `FirstCharToUpper` when we are setting the actual label text.